updated readme (#41)
jjauzion/ws-backend: README.md

  }
}
```

> - if you send the request like this you will get a `403` error which means you are not authenticated (WIP at the moment only create task requires authentication)

Author: ```suggestion
- if you send the request like this you will get a `403` error because you are not authenticated (WIP at the moment only create task requires authentication)
```

  }
}
```

> - if you got a `403` it's because you forgot to add the `auth` Header to your request (see previous chapter)

Reviewer: alors 403 c'est beaucoup plus general. Ca peut-être un token expiré, un user inexistant, un token corrompu, et tout en fait. Il faut partir du principe que quand une connection n'est pas autorisé on ne doit donner aucune information supplémentaires.

> - if you got a `403` it's because you forgot to add the `auth` Header to your request (see previous chapter)

Author: ```suggestion
- if you got a `403` error, check you didn't forget to the `auth` Header in your request (see previous chapter)
```
